javascript/net/grpc/web/generator/grpc_generator.cc

```diff
@@ -1478,8 +1470,7 @@ string GeneratorOptions::OutputFile(const string& proto_file) const {
     string directory;
     string basename;
     PathSplit(proto_file, &directory, &basename);
-    return directory + UppercaseFirstLetter(StripProto(basename)) +
-           "ServiceClientPb.ts";
+    return directory + StripProto(basename) + "_grpc_web_pb.ts";
```

Comment:
We have some unit tests like the [eval_test](https://github.com/grpc/grpc-web/blob/1efe741695e7d330a4a56fe2e223fc027f83705c/packages/grpc-web/test/eval_test.js#L60) and [tsc_test](https://github.com/grpc/grpc-web/blob/1efe741695e7d330a4a56fe2e223fc027f83705c/packages/grpc-web/test/tsc_test.js#L103) that you'd probably need to update.

I wonder if you're able to run the [`run_basic_test.sh`](https://github.com/grpc/grpc-web/blob/master/scripts/run_basic_tests.sh) script locally for testing?